tests/main.test.ts

```diff
@@ -480,6 +486,16 @@ it(`should support disabling the network accesses from the environment`, async (
 });
 
 describe(`read-only and offline environment`, () => {
+  let home: PortablePath;
+  beforeEach(() => {
+    home = npath.toPortablePath(process.env.COREPACK_HOME!);
+  });
+
+  afterEach(async () => {
```

Comment:
```suggestion
  afterEach(async () => {
    const home = npath.toPortablePath(process.env.COREPACK_HOME!);

```

Reply:
Oh yeah, this doesn’t make sense. I think I wanted to reuse home initially. Fixed in https://github.com/nodejs/corepack/pull/662/commits/da69ce3d4c579290a8e262350ed955858e586183